scripts/clean.py

```diff
@@ -25,6 +25,8 @@
 
 CURR_DIR = os.path.abspath(os.getcwd())
 OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+CLOUD_DATASTORE_EMULATOR_DATA_DIR = (
+    os.path.join(CURR_DIR, '.config', 'gcloud', 'emulators', 'datastore'))
```

Comment:
Bit worried about this. We're adding a .config in the oppia/ directory? It will get deployed...

Why not in ../oppia_tools, if it's just for local development?

Reply:
Done.